src/logger/manager.rs

```diff
@@ -1,43 +1,52 @@
-use log::*;
-use std::io::Write;
-
 use crate::cli;
-use env_logger::fmt::{Color, Style, StyledValue};
 
-// Change style based on the message log level
-fn colored_level<'a>(style: &'a mut Style, level: Level) -> StyledValue<'a, &'static str> {
-    match level {
-        Level::Trace => style.set_color(Color::Magenta).value("TRACE"),
-        Level::Debug => style.set_color(Color::Blue).value("DEBUG"),
-        Level::Info => style.set_color(Color::Green).value("INFO "),
-        Level::Warn => style.set_color(Color::Yellow).value("WARN "),
-        Level::Error => style.set_color(Color::Red).value("ERROR"),
-    }
-}
+use tracing::*;
+use tracing_log::LogTracer;
+use tracing_subscriber::{fmt, layer::SubscriberExt, prelude::*, EnvFilter, Layer};
 
 // Start logger, should be done inside main
 pub fn init() {
-    if cli::manager::is_verbose() {
-        std::env::set_var("RUST_LOG", "debug");
-    }
+    // Redirect all logs from libs using "Log"
+    LogTracer::init_with_filter(tracing::log::LevelFilter::Trace).expect("Failed to set logger");
```

Comment:
Is that line necessary ? Are we not changing the filter on the next line ?

Reply:
Nope, here we are configuring the middleware to redirect logs from any library built using the log crate, this filter is different from the next ones, although Trace is not really filtering anything, just trying to be clearer about what level is being redirected (any level).